keps/sig-network/1672-tracking-terminating-endpoints/kep.yaml

```diff
@@ -20,17 +20,18 @@ see-also:
 replaces: []
 
 # The target maturity stage in the current dev cycle for this KEP.
-stage: beta
+stage: stable
 
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.24"
+latest-milestone: "v1.26"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
   alpha: "v1.20"
   beta: "v1.22"
+  stable: "v1.26"
```

Comment:
Comment regarding PRR in the KEP;

> Not yet, but manual upgrade and rollback testing will be done prior to graduating the feature to Beta.

Have those been done? Can you please summarize the setup in which it was done and findings?
If not - can you please do that now?

Reply:
Thanks, I added a new section for manual steps taken for testing rollback. Let me know if they are sufficient.

Reply:
Yes - this looks great - thanks!